tests/test_coordinates.py

```diff
@@ -269,88 +269,57 @@ def test_show():
-                        # check point
-                        npt.assert_allclose(p.flatten(), p_out, atol=1e-15)
-                        # check if system was converted
-                        assert c._system["domain"] == domain_out
-                        assert c._system["convention"] == convention_out
-
-
-def test_multiple_getter_with_conversion():
-    """Test output of 500 random sequential conversions."""
-    # test N successive coordinate conversions
-    N = 500
-
-    # get list of available coordinate systems
-    coords = Coordinates()
-    systems = coords._systems()
-
-    # get reference points in cartesian coordinate system
-    points = ['positive_x', 'positive_y', 'positive_z',
-              'negative_x', 'negative_y', 'negative_z']
-    pts = np.array([systems['cart']['right'][point] for point in points])
-
-    # init the system
-    coords.set_cart(pts[:, 0], pts[:, 1], pts[:, 2])
-
-    # list of domains
-    domains = list(systems)
-
-    for ii in range(N):
-        # randomly select a coordinate system
-        domain = domains[np.random.randint(len(domains))]
-        conventions = list(systems[domain])
-        convention = conventions[np.random.randint(len(conventions))]
-        # convert points to selected system
-        pts = eval(f"coords.get_{domain}('{convention}', convert=True)")
-        # get the reference
-        ref = np.array([systems[domain][convention][point]
-                        for point in points])
-        # check
-        npt.assert_allclose(pts, ref, atol=1e-15)
-        # print
-        print(f"Tolerance met in iteration {ii}")
+    for convention_in in list(systems[domain_in]):
+        for convention_out in list(systems[domain_out]):
+            # for debugging
+            print(f"{domain_in}({convention_in}) -> "
+                  f"{domain_out}({convention_out}): {point}")
+            # in and out points
+            p_in = systems[domain_in][convention_in][point]
+            p_out = systems[domain_out][convention_out][point]
+            # empty object
+            c = Coordinates()
+            # --- set point ---
+            eval(f"c.set_{domain_in}(p_in[0], p_in[1], p_in[2], \
+                    '{convention_in}')")
+            # check point
+            p = eval(f"c.get_{domain_in}('{convention_in}')")
+            npt.assert_allclose(p.flatten(), p_in, atol=1e-15)
+            # --- test without conversion ---
+            p = eval(f"c.get_{domain_out}('{convention_out}')")
+            npt.assert_almost_equal(
+                systems['cart']['right'][point], c.cartesian.flatten())
+            # check internal and returned point
+            # npt.assert_allclose(
+            #     c.cartesian.flatten(), p_in, atol=1e-15)
+            npt.assert_allclose(p.flatten(), p_out, atol=1e-15)
+            # check if system was converted
+            # assert c._system["domain"] == domain_in
+            # assert c._system["convention"] == convention_in
```

Comment:
See above: I'd suggest to keep track of these changes via Git and just remove them instead of commenting.